plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -862,5 +876,18 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		return builder;
 	}
 
+		<#if data.spawnInRaids>
+		@Override public boolean isAlliedTo(Entity ent) {
+		if (super.isAlliedTo(ent)) {

```

Comment:
You should add a procedure condition here to allow more customization (only available if raids are enabled).

Reply:
I know it's beyond my capabilities for the moment, I'll try to do it

Reply:
I removed it, since the option doesn't belong to the raid but to the entities